### PR TITLE
add power attack feat slider / standardize UI

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

**Review by ChrisPJohn — CHANGES_REQUESTED** [review]
The reason I didn't just add a slider before is the actual text of the feat is hard coded to -3/+6 and that text should be updated with the changing values.

**Comment by ThyWoof** [comment]
@ChrisPJohn , changed all descriptions to be dynamic. Set restart_required. It might be possible on an upcoming release to remove this restart dependency.

File: SolastaCommunityExpansion/Viewers/Displays/FeatsDisplay.cs [hunk 1]
```diff
@@ -21,14 +21,20 @@ internal static void DisplayFeats()
             // todo make the acehigh power attack feats tunable here. It is already in the settings (FeatPowerAttackModifier),
             // but the text does not currently update to reflect the actual tuning.
 
-            //UI.Label("");
-            //UI.Label("Settings: ".yellow() + reqRestart);
+            UI.Label("");
+            UI.Label("General:".yellow());
+
+            intValue = Main.Settings.FeatPowerAttackModifier;
+            if (UI.Slider("Power Attack modifier ".white() + reqRestart, ref intValue, 1, 6, 3, ""))
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Viewers/Displays/FeatsDisplay.cs:28]: Does it actually require restart though?

**Review by ChrisPJohn — APPROVED** [review]

**Comment by ThyWoof** [comment]
it was closed by mistake.

**Pull request merged**